app/assets/javascripts/admin/addon/models/admin-plugin.js

```diff
@@ -24,8 +24,8 @@ export default class AdminPlugin {
     return this.name.replaceAll("-", "_");
   }
 
-  get kebabCaseName() {
-    return this.name.replaceAll(" ", "-").replaceAll("_", "-");
+  get dasherizedName() {
+    return dasherize(this.name);
```

Comment:
This was proposed by Joffrey the other day and CVX found the lib name. I guess kebab case does sound mildly offensive  :)